dev/validate-examples/main.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"time"
+
+	yaml "gopkg.in/yaml.v2"
+)
+
+type Config struct {
+	ValidationCmd  string `yaml:"validation_cmd"`
+	TeardownCmd    string `yaml:"teardown_cmd"`
```

Comment:
instead of TeardownCmd you can use `exec.Cmd.Process` to just kill the process :)

Reply:
I would argue that `docker-compose down` is slightly better because:

- it removes stopped containers
- it removed network created for purpose of  corresponding `docker-compose up`

I mean - sure why not - but the way it is done right now leaves workspace cleaner. I am not sure we care that much but it is something to consider.

Reply:
ok, true :)